### ISSUE TITLE
Test-dnssec.t fails

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-engine#670]
When running the unit tests in the current develop branch (6986f7a) this failure is encountered:

```
===(    1428;155  162/?  23/? )=========================================
#   Failed test 'afnic.fr gives ALL_ALGO_SIGNED'
#   at t/Test-dnssec.t line 19.
#   Failed test 'afnic.fr does not give RRSIG_BROKEN'
#   at t/Test-dnssec.t line 30.
===(    1452;162  178/?  31/? )=========================================# Looks like you failed 2 tests of 189.
t/Test-dnssec.t ........... Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/189 subtests 
	(less 2 skipped subtests: 185 okay)
```

I've seen this on Travis and also locally on Debian 10 when installing the preliminary dist tarball using `cpanm`.

**Comment by mattias-p** [comment]
The mentioned message tags are emitted by DNSSEC13.

**Comment by mattias-p** [comment]
I tried to re-record the data and that makes these particular tests pass, but it also breaks a couple of others tests in Test-dnssec.t:
```
...
not ok 150 - dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_NOT_SIGNED
#   Failed test 'dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_NOT_SIGNED'
#   at t/Test-dnssec.t line 19.
not ok 151 - dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_SIGNATURE_NOT_OK
#   Failed test 'dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_SIGNATURE_NOT_OK'
#   at t/Test-dnssec.t line 19.
...
not ok 164 - afnic.fr does not give NO_RESPONSE
#   Failed test 'afnic.fr does not give NO_RESPONSE'
#   at t/Test-dnssec.t line 30.
...
```

**Comment by mattias-p** [comment]
@vlevigneron Could you take a look at this?

**Comment by vlevigneron** [comment]
Yes I know, I'm working on it. I'm on my way to "fix" broken zones.

I'll propose a PR today for that.

le 22 nov., Mattias Päivärinta a ?crit :
> I tried to re-record the data and that makes these particular tests pass, but it also breaks a couple of others tests in Test-dnssec.t:
> ```
> ...
> not ok 150 - dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_NOT_SIGNED
> #   Failed test 'dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_NOT_SIGNED'
> #   at t/Test-dnssec.t line 19.
> not ok 151 - dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_SIGNATURE_NOT_OK
> #   Failed test 'dnssec08-dnskey-not-signed.zut-root.rd.nic.fr gives DNSKEY_SIGNATURE_NOT_OK'
> #   at t/Test-dnssec.t line 19.
> ...
> not ok 164 - afnic.fr does not give NO_RESPONSE
> #   Failed test 'afnic.fr does not give NO_RESPONSE'
> #   at t/Test-dnssec.t line 30.
> ...
> ```
> 
> -- 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/zonemaster/zonemaster-engine/issues/670#issuecomment-557476026

-- 
	Vincent Levigneron  A.F.N.I.C.  Vincent.Levigneron@afnic.fr

**Comment by vlevigneron** [comment]
@mattias-p I just created a PR for that

**Comment by matsduf** [comment]
Resolved by #671
